deploy/examples/cert-manager/README.md

```diff
@@ -0,0 +1,35 @@
+# Steps
+
+## Install cert-manager
+    helm install \
+        cert-manager jetstack/cert-manager \
+        --namespace cert-manager \
+        --create-namespace \
+        --version v1.6.1 \
+        --set installCRDs=true
+
+## Prepare istio-system Namespace
+
+1. Create system namespace:
+
+        kubectl create ns istio-system
+
+2. Provision certificates:
+
+        kubectl apply -f selfsigned-ca.yaml -n istio-system
+
+3. Install cert-manager istio-csr Service:
+
+        helm install -n cert-manager cert-manager-istio-csr jetstack/cert-manager-istio-csr -f deploy-prototype/istio-csr-helm-values.yaml"
```

Comment:
`Error: template: cert-manager-istio-csr/templates/serviceaccount.yaml:3:7: executing "cert-manager-istio-csr/templates/serviceaccount.yaml" at <ne .Values.image.pullSecretName "">: error calling ne: incompatible types for comparison`


Output of this command, please can you check and let me know the exact step.